### PR TITLE
Download document test

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):

- Automated tests for document download from frontend [FLA-818](https://justice.gov.bc.ca/jira/browse/FLA-818)

## Type of change

- [x] New test(non-breaking test)

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

- Tested locally and Github actions

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ export default function DocumentList({
                 <span
                   className="file-href"
                   role="button"
+                  data-testid="file-to-download"
```
Comment (alexjoybc) [inline src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js:77]:
```suggestion
                  data-testid="btn-download-document"
```

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/page/PackageReviewPage.java [hunk 2]
```diff
@@ -26,6 +26,9 @@
     @FindBy(id = "uncontrolled-tab-tab-documents")
     private WebElement documentsTab;
 
+    @FindBy(xpath = "//*[@data-testid='file-to-download']")
```
Comment (alexjoybc) [inline tests/src/test/java/ca/bc/gov/open/jag/efiling/page/PackageReviewPage.java:29]:
```suggestion
    @FindBy(xpath = "//*[@data-testid='btn-download-document']")

```

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/page/PackageReviewPage.java [hunk 3]
```diff
@@ -61,6 +64,15 @@
         return packageDetails;
     }
 
+    public void clickToDownloadDocument() throws InterruptedException {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@data-testid='file-to-download']")));
```
Comment (alexjoybc) [inline tests/src/test/java/ca/bc/gov/open/jag/efiling/page/PackageReviewPage.java:68]:
```suggestion
        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@data-testid='btn-download-document']")));

```

**Review by alexjoybc — CHANGES_REQUESTED** [review]
quick change on the id

**Review by TayGov — APPROVED** [review]

**Pull request merged**